webapp/portlet/src/main/webapp/vue-apps/common/components/RichEditor.vue

```diff
@@ -213,13 +213,14 @@ export default {
         removePlugins = `${removePlugins},tagSuggester`;
       }
 
-      extraPlugins = `${extraPlugins},emoji,formatOption`;
+      extraPlugins = `${extraPlugins},emoji,formatOption,attachImage`;
```

Comment:
To be added in CKEditor instances instead

```suggestion
      extraPlugins = `${extraPlugins},emoji,formatOption`;
```